packages/@azure/storage/.gitignore

```diff
@@ -57,8 +57,15 @@ typings/
 /blob/typings/
 /blob/browser/
 /blob/dist-*/
-/blob/temp
-
+/blob/temp/
```

Comment:
Considering these different libraries are all built uniformly, you could consider having just one set of rules of the form ```
```
/*/temp
/*/dist-*
...
```